llvm/lib/Analysis/InstructionSimplify.cpp

```diff
@@ -4618,6 +4673,9 @@ static Value *simplifySelectWithICmpCond(Value *CondVal, Value *TrueVal,
     }
   }
 
+  if (Value *V = simplifySelectBitTestSpec(CmpLHS, CmpRHS, TrueVal, FalseVal))
+    return V;
+
```

Comment:
We should see test changes in this commit.

Reply:
Please follow @goldsteinn's comment to show test changes in the second commit.